### PR TITLE
Add query_params(query::AbstractString)

### PR DESCRIPTION
**Pull request by EricForgy**
I need this to handle HTML form button "submit"s where the query "username=uname&password=psw" is sent in the body of a POST.

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -43,6 +43,7 @@ function query_params(uri::URI)
     end
     d
 end
+query_params(uri::URI) = query_params(query::AbstractString)
```
Comment (malmaud) [inline src/utils.jl]: Do you just want `query_params(uri.query)` here?

Comment (EricForgy) [inline src/utils.jl]: Oops. Yeah. Sorry. Fixed it now.

**Comment by EricForgy** [comment]
Is this PR ok? I don't know why the Travis test is failing, but I doubt it has to do with this change. 

I am working on a package dealing with OAuth stuff (learning as I go) and it relies on my fork of URIParser. I'd rather depend on an official version so might need to consider another solution if this doesn't get merged.

Thoughts?

Thanks

**Comment by wildart** [comment]
Can you provide a test for this change?

**Comment by EricForgy** [comment]
Sure. Will do tomorrow. Just hitting the sack now (Hong Kong time).

Thanks.

**Comment by malmaud** [comment]
Sorry for this dropping off my radar - I'll merge as soon as the test is committed.

**Comment by EricForgy** [comment]
Thanks @malmaud . I didn't get to the tests yet and am headed home for the day. Try again tomorrow :pray:

**Comment by malmaud** [comment]
Actually this is pretty trivial, I'll just merge.

**Pull request merged**